core/extension/gdextension.h

```diff
@@ -136,6 +136,14 @@ class GDExtension : public Resource {
 		INITIALIZATION_LEVEL_EDITOR = GDEXTENSION_INITIALIZATION_EDITOR
 	};
 
+protected:
+#ifndef DISABLE_DEPRECATED
+	Error _open_library_bind_compat_88418(const String &p_path, const String &p_entry_symbol);
+	void _close_library_bind_compat_88418();
+	void _initialize_library_bind_compat_88418(InitializationLevel p_level);
+	static void _bind_compatibility_methods();
+#endif
```

Comment:
```suggestion
#endif

```

Reply:
Same, the style checks [forced me to remove the blank line after #endif](https://github.com/godotengine/godot/actions/runs/8935217873/job/24543349469)

Reply:
I don't see that in the ci

Reply:
Ah I meant to link this one:
https://github.com/godotengine/godot/actions/runs/8935205337/job/24543315445

I wonder if the issue is I put the blank line after public: instead of between #endif and public:

Reply:
Yes that's not the same as I suggested, normally there should be one between any code and any access specifier, might not work for `endif` though